src/ansys/fluent/core/workflow.py

```diff
@@ -319,21 +319,58 @@ def get_idx(self) -> int:
         """
         return int(self.get_id()[len("TaskObject") :])
 
+    def _populate_duplicate_task_list(self):
+        disp_text = self.display_name()
+        if disp_text.split()[-1].isdigit():
+            new_task = "".join(disp_text.rsplit(f" {disp_text.split()[-1]}", 1))
+            if (
+                new_task
+                == self._command_source._help_string_display_text_map[self._python_name]
+            ):
+                self._python_name = self._python_name + f"_{disp_text.split()[-1]}"
+                self._command_source._help_string_display_text_map[
+                    self._python_name
+                ] = disp_text
+                self._command_source._repeated_task_help_string_display_text_map[
+                    self._python_name
+                ] = disp_text
+
     def python_name(self) -> str:
-        """Get the Pythonic name of this task, as it is in the underlying application.
+        """Get the Pythonic name of this task, from the underlying application.
```

Comment:
```suggestion
        """Get the Pythonic name of this task from the underlying application.
```
Remove the comma in all occurrences, please. Sorry if I wasn't clear on this earlier.